boards/nucleo-f103/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE =$(BOARD)_base
```

Comment:
Could you please change this to `MODULE = board` in reference to upcoming #4376?